app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -837,6 +843,14 @@ public void onPageStop(GeckoSession aSession, boolean b) {
         }
 
         state.mIsLoading = false;
+        long elapsedLoad = SystemClock.elapsedRealtime() - startLoadTime;
+        if (elapsedLoad > MIN_LOAD_TIME
+                && !state.mUri.equals(getHomeUri())
+                && !state.mUri.equals(HOME_WITHOUT_REGION_ORIGIN)
```

Comment:
we don't want to track the Home page? is there a reason why?

Reply:
agree!